components/brave_welcome_ui/components/welcome/index.tsx

```diff
@@ -36,44 +40,57 @@ function Welcome () {
     if (!ref.current) return
     if (!shouldPlayAnimations) return
 
-    const logoAnim: Animation = ref.current.animate({
-      transform: 'translateY(-20px)',
-      filter: 'drop-shadow(7px 2px 5px rgba(14, 1, 41, 0.2)) drop-shadow(14px 3px 10px rgba(32, 5, 89, 0.3)) drop-shadow(20px 3px 15px rgba(37, 7, 87, 0.2))  drop-shadow(25px 5px 30px rgba(25, 3, 73, 0.1)) drop-shadow(50px 4px 50px rgba(19, 3, 40, 0.1))'
-    }, { duration: 1000, fill: 'forwards', easing: 'ease-out' })
+    const logoBoxEl = ref.current.querySelector('.view-logo-box')
+    const backdropEl = ref.current.querySelector('.view-backdrop')
+    const contentEl = ref.current.querySelector('.view-content')
+
+    const s1 = new WebAnimationPlayer()
+
+    s1.to(logoBoxEl, {
+        transform: 'translateY(-20px)',
+        filter: 'drop-shadow(7px 2px 5px rgba(14, 1, 41, 0.2)) drop-shadow(14px 3px 10px rgba(32, 5, 89, 0.3)) drop-shadow(20px 3px 15px rgba(37, 7, 87, 0.2))  drop-shadow(25px 5px 30px rgba(25, 3, 73, 0.1)) drop-shadow(50px 4px 50px rgba(19, 3, 40, 0.1))'
+        }, { fill: 'forwards', easing: 'ease-out' })
+      .to(backdropEl, { scale: 1, opacity: 1 }, { duration: 250, delay: 200, easing: 'ease-out' })
+      .to(contentEl, { transform: 'translateY(0px)', opacity: 1 }, { duration: 250, delay: 200, easing: 'ease-out' })
+
```

Comment:
nit: do you think it would make sense to export these properties from the style.ts file?

Reply:
No, I think this should be in the same file. It's just easier to maintain.